### ISSUE TITLE
Inference with Mask R-CNN .pth file without detectron2 installation using Python

### ISSUE THREAD

**Issue by ClaudiaRaffaelli** [facebookresearch/detectron2#2439]
Hello, I would like to know if it is possible to use a trained Mask R-CNN model, that is a .pth file, obtained using detectron2 on a machine that does not have a detectron2 installation. 
My goal is to make inference with it by using Python. I've already looked into converting a PyTorch model into a Torch Script via Tracing / Annotation or with Caffe2-tracing but I haven't really understood if what I'm trying to do is supported or not.
Thanks.

**Comment by ppwwyyxx** [comment]
There is already an example that does this, except not using python: https://github.com/facebookresearch/detectron2/tree/master/tools/deploy
